### ISSUE TITLE
simple_feature requires mason version 0.1.0-dev.49, version solving failed.

### ISSUE THREAD

**Issue by bazl-E** [RyanDsilva/feature_folder_cli#6]
getting the following error while generating feature structure 

```
bazi@Muhammeds-Mac-mini project-flutter % ff generate -n login -t simple            

Time Taken: 14 milliseconds

✓ Feature login created successfully
✖ The current mason version is 0.1.0-dev.50.
Because simple_feature requires mason version 0.1.0-dev.49, version solving failed.
```
<img width="637" alt="Screenshot 2023-07-12 at 10 02 16 AM" src="https://github.com/RyanDsilva/feature_folder_cli/assets/84278736/f5cb1cf5-ea58-4571-be15-06464f8091ca">

**Comment by alahalil** [comment]
+1

**Comment by pdalfarr** [comment]
same here..

**Comment by wrtisans** [comment]
+1

**Comment by RyanDsilva** [comment]
Solved with #7
